src/providers/caching-token-provider.ts

```diff
@@ -130,6 +136,15 @@ export const CACHE_SEED_TOKENS: {
     WBTC: WBTC_MOONBEAM,
     WGLMR: WRAPPED_NATIVE_CURRENCY[ChainId.MOONBEAM],
   },
+  [ChainId.BSC]: {
+    USDC: USDC_BSC,
+    USDT: USDT_BSC,
+    BUSD: BUSD_BSC,
+    ETH: ETH_BSC,
+    DAI: DAI_BSC,
+    BTC: BTC_BSC,
+    WGLMR: WRAPPED_NATIVE_CURRENCY[ChainId.MOONBEAM],
```

Comment:
chainId.BSC

Reply:
also wtf is wglmr

Reply:
actually why even wglmr here?

Reply:
i guess -> WBNB: ...[ChainId.BSC]

Reply:
Ah guess they just copied and pasted ... my bad should have caught this